internal/services/containers/client/client_gen.go

```diff
@@ -6,6 +6,9 @@ import (
 	"github.com/hashicorp/terraform-provider-azurerm/internal/common"
 )
 
+type autoClient struct {
+}
+
```

Comment:
this file's generated so this'll get reverted?

```suggestion
```

Reply:
whoops this is from some other work